README.md

```diff
@@ -228,7 +228,7 @@ Propagate dependencies and output from `main` sourceSet.
 
 ```groovy
 dependencies {
-    benchmarksCompile sourceSets.main.output + sourceSets.main.runtimeClasspath 
+    add("benchmarksImplementation", sourceSets.main.output + sourceSets.main.runtimeClasspath)
```

Comment:
I know this is not Groovy idiomatic but this syntax should work in both Groovy and Kotlin scripts